editor/src/components/canvas/canvas-strategies/strategies/grid-rearrange-move-strategy.spec.browser2.tsx

```diff
@@ -125,16 +150,13 @@ export var storyboard = (
           data-uid='aaa'
           data-testid='aaa'
         />
-        <Placeholder
```

Comment:
was the fact that this is a Placeholder causing a test issue, or is this just an aesthetic fix?

Reply:
I replaced it with a div because Placeholders don't pass down the `data-testid` attribute